components/yaml/introduction.rst

```diff
@@ -219,6 +222,49 @@ representation to the inline one:
         foo: bar
         bar: baz
 
+Indentation
+"""""""""""
+
+By default the YAML component will use 4 spaces for indentation, this can be
+changed using the second argument as follows::
+
+    // use 2 spaces for indentation
+    echo $dumper->dump($array, 2, 2);
```

Comment:
We should only document the static API:

```php
echo Yaml::dump($array, 2, 2);
```

Can you change that everywhere in this file?